From 86dd09dcd97dc80df5919d1a7285011217daae99 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Mon, 19 Jun 2023 09:14:54 +0530
Subject: [PATCH] Fixed Boot Issue in First Stage Init.

During first stage init boot, it tries to symlink between /system/bin/snapuserd
and /first_stage_ramdisk/system/bin/snapuserd and it fails because
it already is creating symlink during build time.

Removing symlink creation at build time.

Tracked-On: OAM-110092
Change-Id: Ib3db21eab03d29ffc7ff6b0c383d96786fbe9ffc
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 fs_mgr/libsnapshot/snapuserd/Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs_mgr/libsnapshot/snapuserd/Android.bp b/fs_mgr/libsnapshot/snapuserd/Android.bp
index 9261482017..749216cdce 100644
--- a/fs_mgr/libsnapshot/snapuserd/Android.bp
+++ b/fs_mgr/libsnapshot/snapuserd/Android.bp
@@ -159,7 +159,6 @@ cc_binary {
     ramdisk_available: true,
     vendor_ramdisk_available: false,
     ramdisk: true,
-    symlinks: ["snapuserd"],
 }
 
 cc_test {
-- 
2.17.1

